src/styles/theme/dropShadow.ts

```diff
@@ -0,0 +1,11 @@
+export const oakAllDropShadows = {
+  grey20: "0 8px 8px rgba(92, 92, 92, 20%)",
```

Comment:
I think we agreed to strip these back to one placeholder drop shadow. Could you do this.